typings/definitions.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path='tsd.d.ts' />
```

Comment:
What's the need of this typing file? My understanding of this structure is that the `typings` directory contains all type definition get from DefinitelyTyped through `tsd`, and the `lib/typings` subdirectory contains our overrides and extenstions, so the `lib/typings/definitions.d.ts` only needs to reference 'typings/tsd.d.ts', right?